llvm/lib/Target/PowerPC/PPC.td

```diff
@@ -329,6 +329,12 @@ def FeatureAIXLocalExecTLS :
                    "Produce a TOC-free local-exec TLS sequence for this function "
                    "for 64-bit AIX">;
 
+def FeatureAIXSharedLibraryTLSModelHeuristic :
+  SubtargetFeature<"aix-shared-library-tls-model-heuristic",
```

Comment:
This should be an optimization, maybe rename to `aix-shared-library-tls-model-opt`, we perform optimization based on some heuristics.

Reply:
I changed the name to `aix-shared-lib-tls-model-opt`. Hope that is fine.